stable/minio/templates/minioconfig_configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}-config-cm
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+{{ toYaml .Values.minio_server_config | indent 2 }}
```

Comment:
I am not sure this is good practice. While reading the entire config file from values.yaml instead of putting it in this file directly and templatizing it....cc @viglesiasce @prydonius